Makefile

```diff
@@ -222,6 +222,10 @@ testnodeconformance: $(BIN)/node16 $(BIN)/node18 $(BIN)/node20 $(BIN)/node21 $(B
 	cd packages/connect-conformance && PATH="$(abspath $(BIN)):$(PATH)" node18 ./bin/connectconformance --mode client --conf conformance-node.yaml -v ./bin/conformancenodeclient
 	cd packages/connect-conformance && PATH="$(abspath $(BIN)):$(PATH)" node20 ./bin/connectconformance --mode client --conf conformance-node.yaml -v ./bin/conformancenodeclient
 	cd packages/connect-conformance && PATH="$(abspath $(BIN)):$(PATH)" node21 ./bin/connectconformance --mode client --conf conformance-node.yaml -v ./bin/conformancenodeclient
+	# Express Server
```

Comment:
Do we want to run the express and fastify tests with multiple versions of Node? If so, not quite sure how that works with `tsx`.

Reply:
OK so:
![image](https://github.com/connectrpc/connect-es/assets/911455/fc96ec2b-d3ad-4960-b8d8-3dab49d285c4)

And further only supports Node v18 and above. So, if we did want to test with multiple versions, we could only do it with 18, 20, and 21 and we'd then have to get a little more involved and creative with how we install Node into `$(BIN)` (maybe rename `node18` to `node` before the test, etc.)